### PR TITLE
add react-highlight and create binding, implement code tab for script

### LINKED ISSUE #192: Data script page: Code tab

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #192

### CODE REVIEW

File: scan/package.json [hunk 1]
```diff
@@ -19,9 +19,11 @@
     "bs-css": "^11.0.0",
     "bs-let": "^0.1.11",
     "bs-moment": "^0.4.5",
+    "bs-react-syntax-highlighter": "^0.2.0",
```
Comment (sorawit) [inline scan/package.json]: Do we use this?

File: scan/package.json [hunk 2]
```diff
@@ -19,9 +19,11 @@
...
     "js-sha256": "^0.9.0",
     "moment": "^2.24.0",
-    "react": "^16.8.0-0",
+    "react": ">= 0.14.0",
```
Comment (sorawit) [inline scan/package.json]: Is this change necessary?

File: scan/src/pages/ScriptIndexPage.re [hunk 3]
```diff
@@ -130,7 +139,54 @@ let make = (~codeHash, ~hashtag: Route.script_tab_t) => {
...
+                   </Col>
+                   <Col size=1.0> <Text value="Rust 1.39.0" size=Text.Lg /> </Col>
+                 </Row>
+               </Col>
+               <Col size=1.0> <div /> </Col>
+             </Row>
+             <VSpacing size=Spacing.lg />
+           </div>
+           <VSpacing size=Spacing.xl />
+           <Text value={j|📄 Cargo.toml|j} size=Text.Lg color=Colors.grayHeader />
```
Comment (sorawit) [inline scan/src/pages/ScriptIndexPage.re]: Let's use image from design, not emoji

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by sorawit — APPROVED** [review]

**Pull request merged**